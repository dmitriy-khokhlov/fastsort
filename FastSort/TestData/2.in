
Помощь
Войти в мой кошелек
 Забыли пароль?
  Войти
Регистрация за минуту
в электронном кошельке №1 в России*

	Создать кошелек
ПОПОЛНЯЙТЕ
кошелек
в терминальной сети
№1 в России*

 
ПЛАТИТЕ
в онлайн-играх
и социальных сетях

 
БУДЬТЕ УВЕРЕНЫ
платежи конфиденциальны 
и надежно защищены

Visa QIWI Wallet – ваш ключ в мир развлечений и покупок онлайн
Всегда под рукой
Сделать платеж легко даже на ходу — в удобном приложении для смартфонов и планшетов
 
 
Кредиты и переводы онлайн
Достаточно пары минут, чтобы погасить кредит или отправить деньги без визита в банк. Переводы между кошельками – моментально и без комиссии
Покупки по всему миру
Вы можете оплатить покупку в любом интернет-магазине электронным кошельком или бесплатной виртуальной картой QIWI Visa Card
Снять наличные — легко
Деньги удобно переводить из кошелька на банковскую карту или отправлять через системы переводов. С пластиковой картой QIWI Visa легко получить наличные в любом банкомате
Платить не пополняя счета
Зарегистрируйте банковскую карту Visa или MasterCard и безопасно расплачивайтесь ею без указания реквизитов карты при покупках
Начать пользоваться Visa QIWI Wallet

	Создать кошелек
© QIWI, 2014
Что такое QIWI
Карта QIWI Терминалов
Способы пополнения
Помощь
iPhone
Android
iPad
Windows Phone
Другие платформы
Бизнесу
Инвесторам
Оферта
КИВИ Банк
Русский
* по данным исследовательской компании ООО «Синовейт Комкон» на ноябрь 2013.
Исследования проводились по критериям узнаваемости и лояльности к бренду   