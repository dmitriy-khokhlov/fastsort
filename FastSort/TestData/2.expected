*
2013.
2014
Android
Card
MasterCard
Phone
QIWI
QIWI,
Visa
Wallet
Windows
iPad
iPhone
©
«Синовейт
БУДЬТЕ
Банк
Бизнесу
Войти
Всегда
Вы
Деньги
Достаточно
Другие
Забыли
Зарегистрируйте
Инвесторам
Исследования
КИВИ
Карта
Комкон»
Кредиты
Начать
ООО
Оферта
ПЛАТИТЕ
ПОПОЛНЯЙТЕ
Переводы
Платить
Покупки
Помощь
Регистрация
России*
Русский
С
Сделать
Снять
Создать
Способы
Терминалов
УВЕРЕНЫ
Что
банк.
банковскую
банкомате
без
безопасно
бесплатной
бренду
в
ваш
визита
виртуальной
всему
даже
данным
деньги
для
ею
за
защищены
и
из
или
интернет-магазине
исследовательской
к
картой
карту
карты
ключ
комиссии
компании
конфиденциальны
кошелек
кошелька
кошельками
кошельке
кошельком
кредит
критериям
легко
лояльности
любом
между
минут,
минуту
мир
миру
можете
мой
моментально
на
надежно
наличные
не
ноябрь
онлайн
онлайн-играх
оплатить
отправить
отправлять
пароль?
пары
переводить
переводов.
переводы
планшетов
пластиковой
платеж
платежи
платформы
по
погасить
под
покупках
покупку
покупок
получить
пользоваться
пополнения
пополняя
при
приложении
проводились
развлечений
расплачивайтесь
реквизитов
рукой
сети
сетях
системы
смартфонов
социальных
счета
такое
терминальной
удобно
удобном
узнаваемости
указания
ходу
через
чтобы
электронном
электронным
–
—
№1
